apps/channels/models.py

```diff
@@ -34,18 +37,24 @@ def form(self):
         return ChannelForm(initial={"platform": self})
 
     def extra_form(self, *args, **kwargs):
-        from apps.channels.forms import TelegramChannelForm, WhatsappChannelForm
+        from apps.channels import forms
 
         match self:
             case self.TELEGRAM:
-                return TelegramChannelForm(*args, **kwargs)
+                return forms.TelegramChannelForm(*args, **kwargs)
             case self.WHATSAPP:
-                return WhatsappChannelForm(*args, **kwargs)
+                return forms.WhatsappChannelForm(*args, **kwargs)
+            case self.FACEBOOK:
+                initial = kwargs.get("initial", {})
+                if "verify_token" not in initial:
+                    initial["verify_token"] = str(uuid.uuid4())
```

Comment:
nit: you can do this with `initial.setdefault("verify_token", str(uuid.uuid4())`